Level.py

```diff
@@ -55,25 +56,35 @@ def define_level_properties(self):
     def get_touch_auth(self):
         for indx in range(self.ymax):
             for indy in range(self.xmax):
-                if (self.matrix[indx][indy][0] <= self.point[0] < self.matrix[indx][indy][2]) and (self.matrix[indx][indy][1] >= self.point[1] > self.matrix[indx][indy][3]):
-                    print(indx,indy,"ok")
-                    return
+                if (self.matrix[indy][indx][0] <= self.point[0] < self.matrix[indy][indx][2]) and (
```

Comment:
Very long, too long I would say thus not easy to read : maybe create variables and then compare them.